pkg/manager/member/utils.go

```diff
@@ -166,17 +168,33 @@ func PdName(tcName string, ordinal int32, namespace string, clusterDomain string
 	return PdPodName(tcName, ordinal)
 }
 
-// CombineAnnotations merges two annotations maps
-func CombineAnnotations(a, b map[string]string) map[string]string {
+// CombineKVMap merges maps to the left
+func CombineKVMap(a map[string]string, maps ...map[string]string) map[string]string {
```

Comment:
It's a common function. Create a new package in `pkg/util` dir and move this func to it.